lib/gui/index.html

```diff
@@ -38,7 +38,7 @@
       <svg-icon path="../../../assets/etcher.svg"
         width="83px"
         height="13px"
-        os-open-external="https://etcher.io"></svg-icon>
+        os-open-external="https://etcher.io?ref=etcher"></svg-icon>
```

Comment:
I guess here we'd need to rather make it ref=footer if we're focusing on what part of Etcher referred the user? Maybe it should be etcher_footer and etcher_update?